packages/venia-ui/lib/components/ProductFullDetail/__tests__/productFullDetail.spec.js

```diff
@@ -118,18 +108,166 @@ const mockConfigurableProduct = {
     ]
 };
 
-test('Configurable Product has correct initial media gallery image count', () => {
-    const { root } = createTestInstance(
-        <WindowSizeContextProvider>
-            <ProductFullDetail product={mockConfigurableProduct} classes={{}} />
-        </WindowSizeContextProvider>
+const mockSimpleProduct = {
+    __typename: 'SimpleProduct',
+    sku: 'SKU123',
+    name: 'Mock Configrable Product',
+    price: {
+        regularPrice: {
+            amount: {
+                currency: 'USD',
+                value: 123
+            }
+        }
+    },
+    categories: [{ id: 1, breadcrumbs: [{ category_id: 2 }] }],
+    description: 'Mock configurable product has a description!',
+    media_gallery_entries: [
+        {
+            label: 'Base Product - Image 1',
+            position: 1,
+            disabled: false,
+            file: '/base/image-1.jpg'
+        },
+        {
+            label: 'Base Product Image 2',
+            position: 2,
+            disabled: false,
+            file: '/base/image-2.jpg'
+        }
+    ]
+};
+const mockHandleAddToCart = jest.fn();
+const mockHandleSelectionChange = jest.fn();
+const mockHandleSetQuantity = jest.fn();
+
+const talonProps = {
+    breadcrumbCategoryId: undefined,
```

Comment:
Should this type be `string|undefined` as well?

Reply:
Addressed in c99b6836307a4ebb96bf6ca5457bf19ad90fe713